mono/metadata/image.c

```diff
@@ -2048,12 +2050,32 @@ mono_image_open_a_lot (const char *fname, MonoImageOpenStatus *status, gboolean
 
 	// Image not loaded, load it now
 	image = do_mono_image_open (fname, status, TRUE, TRUE, refonly, FALSE, load_from_context);
+	if (problematic)
+		*problematic = TRUE;
 	if (image == NULL)
 		return NULL;
 
 	return register_image (image);
 }
 
+MonoImage *
+mono_image_open_a_lot (const char *fname, MonoImageOpenStatus *status, gboolean refonly, gboolean load_from_context)
+{
+	return mono_image_open_a_lot_parameterized (fname, status, refonly, load_from_context, NULL);
+}
+
+gboolean
+mono_is_problematic_file (const char *fname)
+{
+	MonoImageOpenStatus status;
+	gboolean problematic;
```

Comment:
It's uninitialized and `mono_image_open_a_lot_parametrized` doesn't assign to it on every codepath.